rocketmq-connect-jms/src/main/java/org/apache/rocketmq/connect/jms/Config.java

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
There are so many rocketmq-connect-jms project file in this repo, it will lead to conflicts, so could you remove the JMS related files and changed to depend on it in pom file.

Reply:
ok